inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_params.h

```diff
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include "cldnn/primitives/convolution.hpp"
```

Comment:
Is it really needed?

Reply:
It's a leftover from the 1st patch set of the PR which is not need anymore. Dropped.